docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-        </tr>
-        <tr>
-          <th>9997</th>
-          <td>90.701909</td>
-          <td>1.307473</td>
-        </tr>
-        <tr>
-          <th>9998</th>
-          <td>122.352818</td>
-          <td>4.338779</td>
-        </tr>
-        <tr>
-          <th>9999</th>
-          <td>136.862764</td>
-          <td>3.626774</td>
-        </tr>
-      </tbody>
-    </table>
-    <p>10000 rows × 2 columns</p>
-    </div>
-
-
-
-.. code:: python
-
-    sns.jointplot(df_synthetic['SBP'], df_synthetic['LDL-C'])
-    rho, p_val = scipy.stats.spearmanr(df_synthetic['SBP'], df_synthetic['LDL-C'])
-    rho
-
-
-
-
-.. parsed-literal::
-
-    0.6797565466113832
-
-
-
-
-.. image:: 2020_02_11a_correlation_and_paf_files/2020_02_11a_correlation_and_paf_13_1.png
-
-
-The anticipated problem with this approach: how does correlation affect the joint PAF?
-======================================================================================
-
-This is not relevant to the material above—using the Gaussian Copula to
-capture the correlation between risk-factor exposure will be fine. The
-problem is when we then layer at RiskEffect component into our model, to
-make the risk have a effect on the incidence or mortality rate of our
-cause model. Without loss of generality, suppose the risk effect will
-change the rate of disease.
-
-We have traditionally handled this by calculating a risk-deleted
-incidence:
-
-.. math::
-
-
-   i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
-
-If we *just* include one risk effect, we should still be ok, even with
-correlation. But if we have two risk effects, we have previously used
-assumption
-
-.. math::
-
-
-   1 - \text{PAF}_{\text{Multiplicative}} = \left(1 - \text{PAF}_{\text{Risk1}}\right)\cdot\left(1 - \text{PAF}_{\text{Risk2}}\right).
-
-The more correlated the risk exposure, the more wrong this assumption.
-
-I propose we continue to assume that the relative risks have a “log
-linear” relationship with incidence:
-
-.. math::
-
-
-   i_{e_1, e_2} = i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2},
+Maintaining the correlation is tricky when the risk exposures for a single simulant change over time.
+In general, we have done this by sampling correlated *percentiles* at initialization and then keeping those *percentiles*
+static throughout a simulant's life.
+These percentiles, also known as propensities, are applied to different distributions as the simulant changes age group,
+which means that a simulant's risk exposure in one age group and the next are perfectly auto-correlated.
 
-where we have written :math:`e_1` and :math:`e_2` as “standardized”
-exposures, after scaling and shifting so that the TMREL is 0.0 and a one
-unit increment gives the relative increase in disease burden reported by
-GBD.
+For example, a simulant might have their propensities for height and weight sampled at birth from a
+multivariate uniform distribution with correlation.
```

Comment:
What you mean here is a [copula](https://en.wikipedia.org/wiki/Copula_(probability_theory)), right?

Generally, "multivariate uniform distribution" means a uniform distribution in the product space (or possibly on a subset of the product space), which always equals the independent coupling of univariate uniforms (unless it's on a subset of the product space, in which case the marginals aren't uniform), so this language might be confusing if people are used to the standard usage.

Perhaps you could say "...height and weight sampled at birth from a bivariate distribution in wich the two components are correlated and are each uniformly distributed (i.e., a bivariate [copula](https://en.wikipedia.org/wiki/Copula_(probability_theory))" or something like that, linking out to the definition of copula. I think it would be good to explicitly mention copulas since that seems to be how statisticians usually talk about coupling two marginal distributions in a non-independent way.

Reply:
Yes, you're absolutely right: I was going for a more layman's-terms version, but what I wrote was not remotely correct. I like your idea of briefly explaining what it is and also introducing the word copula.